presto-tests/src/main/java/com/facebook/presto/tests/TestedFeature.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.tests;
+
+import com.google.common.collect.ImmutableSet;
+
+import java.util.Set;
+
+import static java.util.Objects.requireNonNull;
+
+public enum TestedFeature
```

Comment:
You introduced a selection of `TestedFeature`s, which you extend in just next commit. What about starting small, i.e. including only what's really necessary?
At first, this would be just `CREATE_VIEW`, then you'd add `ADD_COLUMN`, `DROP_COLUMN`, etc.